docs/parameters.md

```diff
@@ -20,6 +20,7 @@ The AWS EBS CSI Driver supports [tagging](tagging.md) through `StorageClass.para
 | "inodeSize"                  |                                                    |         | The inode size to use when formatting the underlying filesystem. Only supported on linux nodes and with fstype `ext2`, `ext3`, `ext4`, or `xfs`.                                                                                                                                                                                                                                               |
 | "bytesPerInode"              |                                                    |         | The `bytes-per-inode` to use when formatting the underlying filesystem. Only supported on linux nodes and with fstype `ext2`, `ext3`, `ext4`.                                                                                                                                                                                                                                                  |
 | "numberOfInodes"             |                                                    |         | The `number-of-inodes` to use when formatting the underlying filesystem. Only supported on linux nodes and with fstype `ext2`, `ext3`, `ext4`.                                                                                                                                                                                                                                                 |
+| "ext4BigAllocClusterSize"    |                                                    |         | The cluster size to use when formatting an `ext4` filesystem with the `bigalloc` option enabled. Only supported on linux nodes and with fstype `ext4`.                                                                                                                                                                                                                              |
```

Comment:
It would be good to include a note here specifying kernel compatibility issues with resizing, etc. From [man7](https://man7.org/linux/man-pages/man5/ext4.5.html):

```
Warning: The bigalloc feature is still under development,
and may not be fully supported with your kernel or may
have various bugs.  Please see the web page
http://ext4.wiki.kernel.org/index.php/Bigalloc for
details.
```